File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-04/plagiarized/L6/08/Kasus4L6.kt
package IR

/**
 * @author B15130F5DDB6B5F1622EF91DAC4C1AAE
 */
object Kasus4L6 {
    @JvmStatic
    fun main(args: Array<String>) {
        println("Miles\t\tKilometers")
        println("-------------------------------")
        showTable(10)
    }

    private fun showTable(jumlahData: Int) {
        for (i in jumlahData downTo 1) {
            //Menampilkan hasil
            val temp = jumlahData + 1 - i
            println("$temp\t\t${convertMilesToKilo(temp)}")
        }
    }

    private fun convertMilesToKilo(miles: Int): Double {
        return miles * 1.609
    }
}

File: datasets/IR-Plag/convertai-original-java-rest-python/case-04/plagiarized/L6/08/Kasus4L6.py
import math

def show_table(num_data):
    print("Miles\t\tKilometers")
    print("-------------------------------")
    for i in range(num_data, 0, -1):
        temp = num_data + 1 - i
        print(f"{temp}\t\t{convert_miles_to_kilo(temp):.3f}")

def convert_miles_to_kilo(miles):
    return miles * 1.609

if __name__ == "__main__":
    show_table(10)

